app/bundles/CategoryBundle/Resources/views/Category/list.html.twig

```diff
@@ -134,7 +134,7 @@
 							</td>
 							<td>
 								{% set color = item.getColor() %}
-								<span class="label label-default pa-10" style="background: {{ '#' not in color ? '#' : ''}}{{ color }};"> </span>
```

Comment:
@andersonjeccel For lines like this, where there's no span with `white-space: nowrap`, you could add the span around the label yourself (see `app/bundles/AssetBundle/Resources/views/Asset/list.html.twig` as an example).

Reply:
Fixed using CSS class instead of inline styles (we should avoid inline styles)